montblanc/impl/rime/tensorflow/config.py

```diff
@@ -349,7 +349,7 @@ def test_sersic_shape(self, context):
         units   = HERTZ),
 
     # Beam cube
-    array_dict('ebeam', ('beam_lw', 'beam_mh', 'beam_nud', 'npol'), 'ct',
+    array_dict('ebeam', ('beam_lw', 'beam_mh', 'beam_nud', 4), 'ct',
```

Comment:
Is it necessary to hard-code this? Does  `npol`  not work?

Reply:
There is an explicit assumption deep in the Cpp ebeams code that requires this axis to be 4